README.md

```diff
@@ -1,2 +1,118 @@
-# juttle-engine
-juttle execution engine
+# Juttle Engine
+
+[![Build Status](https://travis-ci.org/juttle/juttle-engine.svg?branch=master)](https://travis-ci.org/juttle/juttle-engine)
```

Comment:
I think we shouldn't release until we resurrect at least one basic functional test and add the lint checks to travis.